public/scss/_forms.scss

```diff
@@ -144,16 +144,18 @@ label:focus ~ .input-help-focused {
   opacity: 1;
 }
 
-.card-icon {
-    opacity: 0;
-    position: absolute;
-    right: 8px;
-    top: 50%;
-    margin-top: -13px;
-    z-index: 10;
-}
+form {
```

Comment:
We won't want to do this as this will mean the card icon stuff will only work inside a form.